changelog/getloc.dd

```diff
@@ -0,0 +1,7 @@
+add `__traits(getLocation, symbol)`
+
+Takes one argument which is a symbol. Returns a `tuple(string, int, int)` whose entries correspond to the filename, line number and column number where the argument
+was declared.
+
+To disambiguate between overloads, pass the result of getOverloads with the desired index, to getLocation.
```

Comment:
This probably needs an example / unittest, since I'm confused as to how to handle overloads after reading it.

Reply:
Done.